Packs/ThreatGrid/Integrations/FeedCiscoSecureMalwareAnalytics/FeedCiscoSecureMalwareAnalytics.py

```diff
@@ -0,0 +1,285 @@
+                        "FirstSeenBySource": obj.get('timestamp')
+                    }
+                }
+            if client.tlp_color:
+                domains_map[current_domain]['fields']['trafficlightprotocol'] = client.tlp_color
+            if client.feed_tags:
+                domains_map[current_domain]['fields']['Tags'] += client.feed_tags
+
+            indicators += set_related_ips(obj, name, client.tlp_color)
+            indicators.append(set_related_files(obj, name, client.tlp_color))
+
+    indicators += list(domains_map.values())
+    return indicators, True
+
+
+def fetch_indicators_command(client: Client, time_to_stop_fetch: datetime = None):
+    """Retrieves indicators from the feed
+
+    Args:
+        client: Client object with request
+        time_to_stop_fetch: Time to stop the fetch before it falls on the docker timeout
+
+    Returns:
+        List. Processed indicator from feed.
+    """
+    current_date = arg_to_datetime('now').date()  # type:ignore
+    while client.first_fetch.date() <= current_date:
+        indicators, status = fetch_indicators(client, time_to_stop_fetch)
+        if not status:
+            return
+        for iter_ in batch(indicators, batch_size=2000):
+            demisto.createIndicators(iter_)
+        # Each request returns one day's indicators, so we'll keep the date of the next day in IntegrationContext.
+        next_fetch = (dateparser.parse('tomorrow', settings={'RELATIVE_BASE': client.first_fetch}))  # type:ignore
+        if client.first_fetch.date() < current_date:
+            demisto.setIntegrationContext({'last_fetch': next_fetch.isoformat()})  # type:ignore
+        client.first_fetch = next_fetch
+        demisto.debug(f'{len(indicators)} XSOAR Indicators were created.')
+    return
+
+
+def get_indicators_command(client: Client, args: Dict[str, str]) -> CommandResults:
+    """Wrapper for retrieving indicators from the feed to the war-room.
+
+    Args:
+        client: Client object with request
+        args: demisto.args()
+
+    Returns:
+        Demisto Outputs.
+    """
+    limit = int(args.get('limit', 50))
+    indicators, status = fetch_indicators(client)
+
+    limit_indicators = indicators[:limit]
+
+    readable_output = tableToMarkdown('CiscoSMA Indicators:', t=limit_indicators, headers=['value', 'type'])
+    command_results = CommandResults(
+        outputs_prefix='CiscoSMA',
+        outputs_key_field='value',
+        outputs=limit_indicators,
+        readable_output=readable_output,
+        raw_response=indicators
+    )
+    return command_results
+
+
+def reset_last_run():
+    """
+    Reset the last run from the integration context
+    """
+    demisto.setIntegrationContext({})
+    return CommandResults(readable_output='Fetch history deleted successfully')
+
+
+def main():
+    """
+    PARSE AND VALIDATE FEED PARAMS
+    """
+    time_to_stop_fetch = arg_to_datetime(arg='in 25 minutes')  # type:ignore # To stop before docker timeout.
```

Comment:
As we are doing here a bit more than just extracting argument values let's move the `try` to cover this logic as well.